bigtable_automation/java/dataflow/pom.xml

```diff
@@ -1,44 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project>
-	<modelVersion>4.0.0</modelVersion>
-	<groupId>org.datacommons</groupId>
-	<artifactId>dataflow</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.datacommons</groupId>
+  <artifactId>dataflow</artifactId>
+  <version>0.0.2-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <properties>
     <beam.version>2.23.0</beam.version>
     <bigtable.version>1.4.0</bigtable.version>
     <slf4j.version>1.7.21</slf4j.version>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
```

Comment:
This dependency is used to build Uber-jars, which is required to package everything into the Docker image.